releasenotes/notes/apm-receiver-timeout-config-6c66ef404b36b766.yaml

```diff
@@ -0,0 +1,14 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    APM: Allow configuring read and write timeouts for the trace-agent HTTP
+    receiver separately using apm_config.receiver_write_timeout and
+    apm_config.receiver_read_timeout.
```

Comment:
If we're listing them in the change log we should also add them in the datadog.yaml.example file.